include/crocoddyl/core/solvers/ipopt/multiple-shooting-nlp.hpp

```diff
@@ -0,0 +1,178 @@
+#ifndef __CROCODDYL_CORE_SOLVERS_IPOPT_MULTIPLE_SHOOTING_NLP_HPP__
+#define __CROCODDYL_CORE_SOLVERS_IPOPT_MULTIPLE_SHOOTING_NLP_HPP__
+
+#include "coin-or/IpTNLP.hpp"
+
```

Comment:
This file doesn't follow our code format.
Please follow the instructions (carefully) in the wiki to automatically format the code.

Reply:
I forgot to format this one. Done